content/docs/2.12/operate/events.md

```diff
@@ -28,3 +28,48 @@ KEDA emits the following [Kubernetes Events](https://kubernetes.io/docs/referenc
 | `TriggerAuthenticationDeleted`        | `Normal`  | When a TriggerAuthentication is deleted                                                                                     |
 | `ClusterTriggerAuthenticationAdded`   | `Normal`  | When a new ClusterTriggerAuthentication is added                                                                            |
 | `ClusterTriggerAuthenticationDeleted` | `Normal`  | When a ClusterTriggerAuthentication is deleted                                                                              |
+
+## CloudEvent Support (Experimental)
+
+### EventSource Resource
+`EventSource` resource now can be created in KEDA for emitting events to user's custom CloudEvent sink. Event will be emitted to both Kubernetes Events and CloudEvents Destination if EventSource resource is created. This specification describes the `EventSource` Custom Resource definition:
+
+[`eventsource_types.go`](https://github.com/kedacore/keda/blob/v1.4.0/pkg/apis/keda/v1alpha1/eventsource_types.go)
+
+```yaml
+apiVersion: keda.k8s.io/v1alpha1
+kind: EventSource
+metadata:
+  name: {cloud-event-name}
+spec:
+  clusterName: {cluster-name} #Optional. Will be used in source/subject 
```

Comment:
We should note that this can be configured in KEDA configuration and this configuration in CRD takes prioritity